app/controllers/workflows_controller.rb

```diff
@@ -86,13 +86,6 @@ def history
 
   delegate :can_update_workflow?, to: :current_ability
 
-  # Fetches the workflow from the workflow service and checks to see if it's active
-  def workflow_active?(wf_name, druid, version)
-    client = WorkflowClientFactory.build
-    workflow = client.workflow(pid: druid, workflow_name: wf_name)
-    workflow.active_for?(version:)
-  end
-
```

Comment:
this method was added to the existing `WorkflowService` class for easy testing and re-use elsewhere